plugins/board-resources/src/components/EditCard.svelte

```diff
@@ -35,15 +61,156 @@
+        <Button icon={IconClose} kind="transparent" size="large" on:click={() => dispatch('close')} />
+      </div>
+      <div class="row">
+        <div class="first-column">
+          <Icon icon={board.icon.Card} size="large" />
+        </div>
+        <div class="title">
+          <EditBox bind:value={object.title} maxWidth="39rem" focus on:change={() => change('title', object?.title)} />
+        </div>
+      </div>
+      <div class="row">
+        <div class="first-column" />
+        <div>
+          <Label label={board.string.InList} /><span class="state-name">{state?.title}</span>
+        </div>
+      </div>
+      <div class="card-info">
+        <div class="left-pane">
+          <div class="row">
+            <div class="first-column" />
+            <CardFields value={object} />
+          </div>
+          <div class="row section-header">
+            <div class="first-column">
+              <Icon icon={board.icon.Card} size="large" />
+            </div>
+            <div class="section-label">
+              <Label label={board.string.Description} />
+            </div>
+          </div>
+          <div class="row">
+            <div class="first-column" />
+            <div class="description">
+              <StyledTextBox
+                alwaysEdit={true}
+                showButtons={false}
+                placeholder={board.string.DescriptionPlaceholder}
+                bind:content={object.description}
+                on:value={(evt) => change('description', evt.detail)}
+              />
+            </div>
+          </div>
+          <!-- TODO attachments-->
+          <!-- TODO checklists -->
+          <div class="row section-header">
+            <div class="first-column">
+              <Icon icon={IconActivity} size="large" />
+            </div>
+            <div class="section-label">
+              <Label label={activity.string.Activity} />
+            </div>
+            <Button
+              kind="no-border"
+              label={isActivityShown ? board.string.HideDetails : board.string.ShowDetails}
+              width="100px"
+              on:click={() => {
+                isActivityShown = !isActivityShown
+              }}
+            />
+          </div>
+          <div class="row">
+            <div class="first-column" />
+            <div class="comment-input">
+              <Component is={chunter.component.CommentInput} props={{ object }} />
+            </div>
+          </div>
+          {#if isActivityShown === true}
+            <Component is={activity.component.Activity} props={{ object, noCommentInput: true, transparent: true }}>
+              <slot />
+            </Component>
+          {/if}
+        </div>
+
+        <div class="right-pane"><CardActions value={object} /></div>
+      </div>
+    </div>
+  </Scroller>
 {/if}
+
+<style lang="scss">
+  .root {
+    display: flex;
+    justify-content: start;
+    flex-direction: column;
+    padding: 20px;
+    width: 650px;
```

Comment:
Please use 'rem' instead of 'px' and check out existing styles to specify width, like

```
.w-full { width: 100%;  }
.w-85 {width: 21.25rem; }
.w-165 {width: 41.25rem; }
```

from _layout.scss